doc/api.rst

```diff
@@ -403,6 +418,122 @@ Computation
 :py:attr:`~core.groupby.DataArrayGroupBy.where`
 :py:attr:`~core.groupby.DataArrayGroupBy.quantile`
 
+
+String manipulation
+-------------------
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor_method.rst
```

Comment:
not sure if it's the template or the `UncachedAccessor` class, but somehow the signature loses the `self`

Reply:
it seems that's the case for all methods, not just the accessors, so nothing to do here